lib/main.py

```diff
@@ -99,6 +99,10 @@ def load_environment():
     else:
         Env.log('WARNING', 'Both stack and HWIMO information are missing')
 
+    if obj_options.str_debug:
+        Env.log('INFO', 'Will stop on error, so Puffer won\'t deconfig a case if it fails, '
```

Comment:
Description doesn't match with implementation. Actually deconfig for current failing case is executed.